Resources/doc/reference/batch_actions.rst

```diff
@@ -189,13 +189,12 @@ granularity), the passed query is ``null``.
             throw new AccessDeniedException();
         }
 
-        $request = $this->get('request');
         $modelManager = $this->admin->getModelManager();
 
-        $target = $modelManager->find($this->admin->getClass(), $request->get('targetId'));
+        $idx = $this->get('request')->get('idx');
 
-        if( $target === null){
-            $this->addFlash('sonata_flash_info', 'flash_batch_merge_no_target');
+        if(empty($idx){
+            $this->getFlashBag()->add('sonata_flash_info', 'flash_batch_merge_no_target');
```

Comment:
`$this->addFlash` instead.

See: https://github.com/sonata-project/SonataAdminBundle/blob/2bb5c41dd77f73f8b449021278dc23dc0826e76e/Controller/CRUDController.php#L1225